internal/pkg/service/stream/definition/by.go

```diff
@@ -0,0 +1,42 @@
+package definition
+
+import (
+	"strconv"
+
+	"github.com/keboola/go-client/pkg/keboola"
+)
+
+const (
+	BySystem = "system"
+	ByUser   = "user"
+)
+
+type ByType string
+
+// By describes actor of an operation.
+type By struct {
+	Type      ByType `json:"type" validate:"required,oneof=system user"`
+	TokenID   string `json:"tokenId,omitempty"`
+	TokenDesc string `json:"tokenDesc,omitempty"`
+	UserID    string `json:"userId,omitempty"`
+	UserName  string `json:"userName,omitempty"`
+}
+
+func (v ByType) String() string {
+	return string(v)
+}
```

Comment:
Should be next to `ByType` definition. It is mixed.

Reply:
https://github.com/keboola/keboola-as-code/pull/1774/commits/1b1248ea0d813c8827a1777b3517e061e0ee4891